### PR TITLE
chore/remove `domtoimage`

### PR DESCRIPTION
**Pull request by liady**
**Problem:**
`domtoimage` caused issues when it was imported as an external inside a web worker.

**Fix:**
The underlying externals issue still remains, but we removed `domtoimage` as we don't use it anymore (and it was our only external dependecy)

### CODE REVIEW

**Review by balazsbajorics — APPROVED** [review]

**Review by Rheeseyb — APPROVED** [review]

File: editor/pnpm-lock.yaml
Comment (seanparsons) [inline editor/pnpm-lock.yaml:1]: Weird, the lock file got much bigger as a result of removing something...

**Review by seanparsons — APPROVED** [review]

File: editor/pnpm-lock.yaml
Comment (balazsbajorics) [reply editor/pnpm-lock.yaml:1]: oh this _is_ weird!

Comment (balazsbajorics) [reply editor/pnpm-lock.yaml:1]: Liad did you run this in our nix environment? maybe you had a different node version or a different `pnpm` version that caused the lockfile to be re-done

**Review by balazsbajorics — CHANGES_REQUESTED** [review]
blocking this until we figure out the huge lockfile change

File: editor/pnpm-lock.yaml
Comment (liady) [reply editor/pnpm-lock.yaml:1]: yes, I ran it in our nix environment, I removed some dependencies so expected a lockfile diff, but you're right that the lockfile result seems weird. Will check it again.

Comment (liady) [reply editor/pnpm-lock.yaml:1]: Seems like it's mostly prettier changes 
image
image

**Review by balazsbajorics — APPROVED** [review]

File: editor/pnpm-lock.yaml
Comment (balazsbajorics) [reply editor/pnpm-lock.yaml:1]: thanks!

**Pull request merged**